llm_unlearn_ucl/unlearn_harm.py

```diff
@@ -248,7 +241,9 @@ def main(args) -> None:
         )
     else:
         model = AutoModelForCausalLM.from_pretrained(
-            args.model_name, cache_dir=args.cache_dir, torch_dtype=torch.bfloat16
+            args.model_name,
+            cache_dir=args.cache_dir,
+            torch_dtype=torch.bfloat16,
```

Comment:
This is also commented out in #117 , as model precision should ideally be defined as an argument.